pytools/datatable.py

```diff
@@ -1,6 +1,11 @@
 from pytools import Record
 
 
+__doc__ = """
+.. autoclass:: DataTable
```

Comment:
This probably needs an `.. automodule:: pytools.datatable` to do something.

Reply:
Thanks, added!